entries/urls.py

```diff
@@ -12,4 +12,10 @@
         name='project-list'),
     url(r'^projects/(?P<pk>\d+)/$', views.ProjectUpdateView.as_view(),
         name='project-detail'),
+    url(
+    	r'^accounts/login/$', auth_views.login, {'extra_context': {'next':'/entries'}}), # Why specifying next doesn't work?
```

Comment:
next is a prameter taken from the URL (http://example.com/login/?next=/clients/) and not part of the template context. The page where Django will redirect to is taken from that parameter or if the value is unsafe or not defined from `settings.LOGIN_REDIRECT_URL`. See Django's source for the exact implementation: https://github.com/django/django/blob/stable/1.8.x/django/contrib/auth/views.py#L39-L53